libsolidity/interface/CompilerStack.cpp

```diff
@@ -547,6 +547,7 @@ bool CompilerStack::analyze()
 		if (noErrors)
 		{
 			ModelChecker modelChecker(m_errorReporter, m_smtlib2Responses, m_modelCheckerSettings, m_readFile, m_enabledSMTSolvers);
+			modelChecker.searchPragma(applyMap(m_sourceOrder, [](auto _source) { return _source->ast; }));
```

Comment:
The type is pretty short here and I think it's easier to read when it's used explicitly. It also makes it more apparent that this should really be a reference.

```suggestion
			modelChecker.searchPragma(applyMap(m_sourceOrder, [](Source const& _source) { return _source->ast; }));
```

Reply:
It's actually `Source const*`, but changed anyway

Reply:
Ah, ok. I should have noticed you're not iterating the original container that holds the structs but another one that has pointers to them. Anyway, all the more reason to show the type :)